xforce_connector.py

```diff
@@ -13,14 +13,17 @@
 # --
 
 import datetime
-from phantom.action_result import ActionResult
```

Comment:
The code change here is mostly suggested by the pre-commit runs.
Additional, 2to3 is run on this file.